commands/utils/batch/batch_runner.js

```diff
@@ -66,7 +66,7 @@ function run_test(payload, env = "prod") {
 
 async function run(lt_config, batches, env, i = 0) {
   totalBatches = batches.length;
-  console.log("Total number of batches " + totalBatches);
+  //console.log("Total number of batches " + totalBatches);
```

Comment:
@japneetlambdatest is this intentional ?

Reply:
@asad9711  yes